src/main/java/com/shilymily/bms/entity/Company.java

```diff
@@ -0,0 +1,32 @@
+package com.shilymily.bms.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+/**
+ * Created by hmaskai on 9/8/18.
+ */
+@Entity
+@Table(name = "companies")
+@Setter @AllArgsConstructor @NoArgsConstructor @ToString
```

Comment:
You can just use @Data and remove all of these.

Reply:
Done